Port Lambda to Scala

LATaylor-guardian opened the pull request:
The following PR ports this lambda over to Scala from Go. This has been done so that going forward it can more easily be maintained and is in a language familiar with the department. 

The following improvements have been made:

- Cloudformed the lambda.
- Provide email lists as environment variables to the lambda. This means that they are now more easily updatable in the AWS console UI.
- Made Deployable with Riff Raff.
- Reduced code footprint.

Commit:
Add new email addresses to receive expired sponsorships email.

kelvin-chappell commented on cloudformation.yaml:
```diff
@@ -0,0 +1,92 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Sends a daily email of all expired and soon to be expiring emails.
```
expiring emails?

kelvin-chappell reviewed (approved):
Nice work! 👍

kelvin-chappell commented:
Is it possible to put something in the description of the Lambda to suggest where its source code is?  So that you can trace it back from the AWS console.

Commit:
Port sponsorship expiry email Lambda over to Scala.

   1. Cloudform.
   2. Make deployable via Riff Raff.
   3. Make email addresses configurable by cloudformation parameters.
